src/main/java/org/spectrumauctions/sats/core/model/gsvm/GSVMBidder.java

```diff
@@ -46,15 +46,32 @@
 
     @Override
     public BigDecimal calculateValue(Bundle bundle) {
-        double value = 0;
+        List<Double> values = new ArrayList<>();
+        int synergyCount = 0;
         for (Good good : bundle.getSingleQuantityGoods()) {
             GSVMLicense license = (GSVMLicense) good;
             if (this.values.containsKey(license.getLongId())) {
-                value += this.values.get(license.getLongId()).doubleValue();
+                values.add(this.values.get(license.getLongId()).doubleValue());
+                synergyCount++;
+            } else if (world.isLegacyGSVM()) {
+                synergyCount++;
             }
         }
+        
+        double value = 0;
+        if(!world.isLegacyGSVM() && this.getSetupType().equals("Regional Bidder Setup")) {
+        	// Only use 4 highest base values for regional bidders
+        	values.sort(Double::compare);
+        	Collections.reverse(values);
+        	value = values.stream().limit(4).mapToDouble(Double::doubleValue).sum();
+        	// Limit synergy to 4 items
+        	synergyCount = Math.min(synergyCount, 4);
+        } else {
+        	value = values.stream().mapToDouble(Double::doubleValue).sum();
+        }
+        
         double factor = 0;
-        if (!bundle.getBundleEntries().isEmpty()) factor = 0.2 * (bundle.getBundleEntries().size() - 1);
+        if (synergyCount > 0) factor = 0.2 * (synergyCount - 1);
```

Comment:
I think the old version can lead to some problems. This way it is clear that no negative values will be returned also for bundles who only contain items with 0 base value

Reply:
I agree! 👍 Thanks for the catch.